docs/table/performance.inc.rst

```diff
@@ -24,3 +24,17 @@ end::
   >>> for i in range(100):
   ...    rows.append((1, 2))
   >>> t = Table(rows=rows, names=['a', 'b'])
+
+Writing a |Table| with |MaskedColumn| to .ecsv using
+:meth:`~astropy.table.Table.write` can be very slow::
+
+    >>> from astropy.table import Table
+    >>> import numpy as np
+    >>> x = np.arange(10000, dtype=float)
+    >>> tm = Table([x], masked=True)
+    >>> tm.write('tm.ecsv', overwrite=True)
```

Comment:
Should we mark this line as SKIP to not waste CI resources on something that is known to be slow?